celery_prometheus_exporter.py

```diff
@@ -17,10 +17,17 @@
 __VERSION__ = (1, 2, 0, 'final', 0)
 
 
+def decode_buckets(buckets_list):
+    return [float(x) for x in buckets_list.split(',')]
+
+
+DEFAULT_BUCKETS = '.005, .01, .025, .05, .075, .1, .25, .5, .75, 1.0, 2.5, 5.0, 7.5, 10.0'
```

Comment:
According to [prometheus_client/metrics.py](https://github.com/prometheus/client_python/blob/3cb4c9247f3f08dfbe650b6bdf1f53aa5f6683c1/prometheus_client/metrics.py#L473) there should also be a bucket for Inf :)

Reply:
Done it, thank you for checking!

Reply:
Thank you :) In prometheus-client 0.3.0 `DEFAULT_BUCKETS` was not available. I think we should simply remove support for anything below prometheus-client 0.5.0. What do you think?

Reply:
Sorry for the late response.
Yes, it's good idea. How can I disable this?

Reply:
Sorry for the late response :( I will try something tomorrow. If it doesn't work then we will have to duplicate the `DEFAULT_BUCKETS` setting for older clients.